Source/LibationFileManager/Templates.cs

```diff
@@ -274,8 +276,10 @@ private static void RemoveSpaces(List<string> parts)
 		{
 			new PropertyTagCollection<LibraryBookDto>(caseSensative: true, StringFormatter)
 			{
-				{ TemplateTags.Title, lb => lb.Title },
+				{ TemplateTags.Title, lb => lb.TitleWithSubtitle },
 				{ TemplateTags.TitleShort, lb => getTitleShort(lb.Title) },
```

Comment:
To maintain compatibility with the old tag, should this get `getTitleShort(lb.TitleWithSubtitle)`? I think it'd end up the same in 90% of cases, but old `lb.Title` was with Subtitle.

Reply:
> old `lb.Title` was with Subtitle.

True, but they were concatenated with a colon so a substring on just the title will always return the title short.